### ISSUE TITLE
10-10d Update school cert guidance

### ISSUE THREAD

**Issue by baltes** [department-of-veterans-affairs/va.gov-team#95221]

**Comment by jamiefiore** [comment]
@baltes we just got approved on the content that needs to go on the upload/confirmation page. 
Here's the [content for the school cert upload page](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=3954-85864&t=WtG1ypNTnAbXNujB-4)
Here's the [content for the confirmation page](https://www.figma.com/design/UmAtr3ULQEInMXfNFwP0g0/Application-for-CHAMPVA?node-id=3954-86270&t=Pl7eDGeLnJ9IvmoH-4) when files are missing.

**Comment by baltes** [comment]
@michaelclement Do we have enough to consider this one ready?

**Comment by michaelclement** [comment]
@baltes yes, apologies, I didn't see the tag earlier.

**Comment by michaelclement** [comment]
Work done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/32963). Waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
@baltes This is ready to review. Please see the screenshots:
| |upload|confirmation|
|-|------|-----|
|Desktop|<img width="498" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3VvQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d2ffe0ded48c30e4ba6b51ef74169ea6ff945781/Screenshot%202024-11-08%20at%2009.47.31.png" alt="Screenshot 2024-11-08 at 09.47.31.png" />|<img width="379" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3lvQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ce96a5974530666b44ac90cc5e0851605b8106d8/Screenshot%202024-11-08%20at%2010.38.29.png" alt="Screenshot 2024-11-08 at 10.38.29.png" />|

**Comment by michaelclement** [comment]
This is visible on staging, btw.

**Comment by baltes** [comment]
@michaelclement Just checked staging, it looks like the copy on the upload page is a little different from Figma. I think it's just the bullets in the first section.

Figma:
<img width="209" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3F6Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d50e620e56ef0d8035d9fa4b9d524386bd0b7ece/image.png" alt="image.png" />

Staging:
<img width="452" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3V6Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d116e10fbf6f0af85bd006be200d71f19a15df9f/image.png" alt="image.png" />

**Comment by michaelclement** [comment]
Ah yes, looks like I missed the "Start and end dates" bullet. I'll get that added in a new PR. Thanks! @baltes

**Comment by baltes** [comment]
@jamiefiore Did you want to take a look at this before we close it out?
